doc/source/user_guide/tutorials/mesh/split_mesh.rst

```diff
@@ -0,0 +1,240 @@
+.. _ref_tutorials_split_mesh:
+
+============
+Split a mesh
+============
+
+:bdg-mapdl:`MAPDL` :bdg-lsdyna:`LSDYNA` :bdg-fluent:`Fluent` :bdg-cfx:`CFX`
+
+.. include:: ../../../links_and_refs.rst
+
+.. |MeshesContainer| replace:: :class:`MeshesContainer <ansys.dpf.core.meshes_container.MeshesContainer>`
+.. |split_mesh| replace:: :class:`split_mesh <ansys.dpf.core.operators.mesh.split_mesh.split_mesh>`
+.. |split_on_property_type| replace:: :class:`split_on_property_type <ansys.dpf.core.operators.scoping.split_on_property_type.split_on_property_type>`
+.. |from_scopings| replace:: :class:`from_scopings <ansys.dpf.core.operators.mesh.from_scopings.from_scopings>`
+.. |ScopingsContainer| replace:: :class:`ScopingsContainer <ansys.dpf.core.scopings_container.ScopingsContainer>`
+.. |PropertyField| replace:: :class:`PropertyField <ansys.dpf.core.property_field.PropertyField>`
```

Comment:
Great job Luisa! You made heavy use of substitution definitions which very nice. 

I observed the following concerning these definitions:
- Some definitions already in links_and_refs.rst are being redefined, for example `|MeshesContainer|` and `|ScopingsContainer|`. If definitions in links_and_refs.rst are changed in the future, the changes will not reflect in your examples because local definitions will take precedence.
- You have same substitution definitions in more than one .rst file, for example `|meshes_provider|` common to _get_mesh_from_result_file.rst_ and _extract_mesh_in_split_parts.rst_. If there is a restructuring of the dpf.core package in the future affecting the location `|meshes_provider|` is pointing to, then each of the definitions in your examples have to be located and changed. It is a similar problem to that above.

The solution to this is to move all your substitution definitions to links_and_refs.rst. Then you have only one source of truth. Let me know what you think about the suggested approach.